### PR TITLE
[KOGITO-3361]  Fix Kogito for PMML native

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @evacchi  @jiripetrlik

See https://issues.redhat.com/browse/KOGITO-3361

This PR add PMML related classes to the KogitoAssetsProcessor, so that they are found during runtime of nativeimage

### CODE REVIEW

**Review by evacchi — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -18,4 +18,4 @@ target/
 .classpath
 .factorypath
 .vscode
-.run
+.run/
```
Comment (danielezonca) [inline .gitignore:21]: Is this change needed?

**Review by danielezonca — APPROVED** [review]
LGTM :+1:

**Review by jiripetrlik — APPROVED** [review]
